memory/timeindex.py

```
@@ -33,8 +33,12 @@ def _ensure_engine_exists(self, lanlan_name: str, db_path: str | None = None) ->
                     db_path = os.path.join(ensure_character_dir(config_mgr.memory_dir, lanlan_name), 'time_indexed.db')
                     logger.info(f"[TimeIndexedMemory] 角色 '{lanlan_name}' 不在配置中，使用默认路径: {db_path}")
 
-            engine = create_engine(f"sqlite:///{db_path}")
-            connection_string = f"sqlite:///{db_path}"
+            # 确保数据库文件的父目录存在
+            os.makedirs(os.path.dirname(db_path), exist_ok=True)
+            # Windows 路径使用反斜杠，SQLite URI 需要正斜杠
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)